openapi/kas-fleet-manager.yaml

```diff
@@ -64,51 +64,51 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/KafkaRequest'
+                $ref: "#/components/schemas/KafkaRequest"
               examples:
                 KafkaRequestGetResponseExample:
-                  $ref: '#/components/examples/KafkaRequestExample'
+                  $ref: "#/components/examples/KafkaRequestExample"
                 KafkaRequestGetResponseWithFailedCreationStatusExample:
-                  $ref: '#/components/examples/KafkaRequestFailedCreationStatusExample'
+                  $ref: "#/components/examples/KafkaRequestFailedCreationStatusExample"
           description: Kafka request found by ID
         "401":
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Error'
+                $ref: "#/components/schemas/Error"
```

Comment:
Can we avoid changing this format or do it in separate PR. It does produce number of conflicts as well as prevent us from generating valid changelog in SDKs

Reply:
Sorry about that. I think my IDE made those changes. I'll reverse it. Thanks @wtrocki